src/components/Map/index.js

```diff
@@ -1,12 +1,32 @@
+import {GeoSearchControl} from 'leaflet-geosearch';
 import isEqual from 'lodash/isEqual';
 import PropTypes from 'prop-types';
-import React, {useEffect} from 'react';
+import React, {useContext, useEffect} from 'react';
+import {defineMessage, useIntl} from 'react-intl';
 import {MapContainer, Marker, TileLayer, useMap, useMapEvent} from 'react-leaflet';
 import {useGeolocation} from 'react-use';
 
+import {ConfigContext} from 'Context';
 import {DEFAULT_LAT_LNG, DEFAULT_ZOOM, MAP_DEFAULTS, TILE_LAYERS} from 'map/constants';
 import {getBEMClassName} from 'utils';
 
+import OpenFormsProvider from './provider';
+
+export const mapSearchControlButtonAreaLabel = defineMessage({
```

Comment:
1. why is this being exported?
2. Typo: areaLabel -> ar**i**aLabel
3. you can also use the `defineMessages` to created an object with your own keys, so you can refer to them in the code with `messages.searchControlLabel`, for example